platform/wab/src/wab/client/components/grouping/VirtualTree.tsx

```diff
@@ -38,59 +40,29 @@ interface TreeRowData<T> {
     matcher: Matcher;
     nodes: LinearTreeNode<T>[];
     renderElement: RenderElement<T>;
-    toggleExpand: (key: Key) => void;
+    selectedIndex?: number;
+    nodeAction?: NodeAction<T>;
+    toggleExpand: (key: NodeKey) => void;
   };
 }
 
 interface VirtualTreeProps<T> {
   rootNodes: T[];
-  query?: string;
-  getNodeKey: (node: T) => Key;
-  getNodeChildren: (node: T) => T[];
-  getNodeSearchText: (node: T) => string;
-  getNodeHeight: (node: T) => number;
   renderElement: RenderElement<T>;
-  defaultOpenKeys?: "all" | Key[];
+  nodeData;
```

Comment:
Missing nodeData type?

Reply:
Oh right, I somehow messed that up when merging useTreeData back to VirtualTree.tsx. I'm used to relying on `noImplicitAny` 😅